lib/stream-chat/client.rb

```diff
@@ -283,6 +285,30 @@ def get_export_channel_status(task_id)
       get("export_channels/#{task_id}")
     end
 
+    def revoke_tokens(before)
+      before = before.iso8601 if before.instance_of?(DateTime)
```

Comment:
we expect `rfc3339`

Reply:
https://apidock.com/ruby/v2_5_5/DateTime/rfc3339
it uses the the iso8601 call only to parse it, since rfc3339 is a profile of ISO-format.
most time libs share the parse the string between the two. Not sure what to take as the standard.
What do you suggest, this has been a confusing point for me :sweat_smile: